podcasts/CustomStepper.swift

```diff
@@ -0,0 +1,188 @@
+import UIKit
+
+class CustomStepper: UIControl {
+    private static let buttonWidth: CGFloat = 44
+    private static let buttonHeight: CGFloat = 44
+
+    private static let initialHoldTime: TimeInterval = 1
+    private static let holdRepetition: TimeInterval = 0.15
+
+    let minusButton = UIButton(type: .custom)
+    let plusButton = UIButton(type: .custom)
+
+    override var tintColor: UIColor! {
+        didSet {
+            minusButton.tintColor = tintColor
+            plusButton.tintColor = tintColor
+        }
+    }
+
+    var minimumValue: Int = 0 {
+        didSet {
+            if currentValue < minimumValue {
+                currentValue = minimumValue
+            }
+        }
+    }
+
+    var maximumValue: Int = 200 {
+        didSet {
+            if currentValue > maximumValue {
+                currentValue = maximumValue
+            }
+        }
+    }
+
+    var bigIncrements: Int = 1
+    var smallIncrements: Int = 1
+    var smallIncrementThreshold: Int = 1
+    var currentValue: Int = 1 {
+        didSet {
+            accessibilityValue = "\(currentValue)"
+        }
+    }
```

Comment:
Instead of generics why not abstract this to a protocol and have two implementations one for Int and another for TimeInterval that you can set on `init`?